.run/Muster Postgres Controller.run.xml

```diff
@@ -0,0 +1,22 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Muster Postgres Controller" type="mocha-javascript-test-runner" nameIsGenerated="true">
```

Comment:
What are these files for? Is this part of the attempt to have separate ITs and UTs?

Reply:
No, these are IntelliJ specific files, just like the `.vscode` folder you can see in this repo. These are stored test run configuration allowing IntelliJ users to conveniently
<img width="763" alt="Screen Shot 2021-07-28 at 12 23 12 PM" src="https://user-images.githubusercontent.com/58491416/127368189-f43999c5-feaf-4be2-9fc8-268d428303c5.png">
 click on a test and run it.